Week1/homework/app.js

```diff
@@ -1,11 +1,171 @@
 'use strict';
 
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+  // *** there are some green wavy lines. those are
+  // because of turkish names of books or authors.
+  const myBooks = [
+    'huzursuzluk',
+    'oliver_twist',
+    'war_and_peace',
+    'the_death_of_ivan_ilyich',
+    'resurrection',
+    'the_da_vinci_code',
+    'think_and_grow_rich',
+    'gone_with_the_wind',
+    'peygamberim',
+    'benim_adim_kirmizi',
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  /* THIS FUNCTION CHANGED ON STEP 1.5
+function makeUlFromMyBooksArray() {
+  const ul = document.createElement('ul');
+  for (let i = 0; i < myBooks.length; i++) {
+    const li = document.createElement('li');
+    li.innerText = myBooks[i];
+    li.id = myBooks[i].slice(0, 6);
+    ul.appendChild(li);
+  }
+  console.log(ul);
+}
+makeUlFromMyBooksArray();
+*/
+
+  const myBooksInfo = {
+    huzursuzluk: {
+      title: 'Huzursuzluk',
+      language: 'Turkish',
+      author: 'zulfu livaneli',
+    },
+    oliver_twist: {
+      title: 'Oliver Twist',
+      language: 'English',
+      author: 'Charles Dickens',
+    },
+    war_and_peace: {
+      title: 'War and Peace',
+      language: 'Russian',
+      author: 'Leo Tolstoy',
+    },
+    the_death_of_ivan_ilyich: {
+      title: 'The Death of Ivan Ilyich',
+      language: 'Russian',
+      author: 'Leo Tolstoy',
+    },
+    resurrection: {
+      title: 'Resurrection',
+      language: 'Russian',
+      author: 'Leo Tolstoy',
+    },
+    the_da_vinci_code: {
+      title: 'The Da Vinci Code',
+      language: 'english',
+      author: 'Dan Brown',
+    },
+    think_and_grow_rich: {
+      title: 'Think and Grow Rich',
+      language: 'English',
+      author: 'Napoleon Hill',
+    },
+    gone_with_the_wind: {
+      title: 'Gone With The Wind',
+      language: 'English',
+      author: 'Margaret Mitchell',
+    },
+    peygamberim: {
+      title: 'Peygamberim',
+      language: 'Turkish',
+      author: 'Salih Suruc',
+    },
+    benim_adim_kirmizi: {
+      title: 'Benim Adim Kirmizi',
+      language: 'Turkish',
+      author: 'Orhan Pamuk',
+    },
+  };
+
+  // books infos appended in ul as li
+  function copyMyBooksInfoToHtml() {
+    const h1 = document.createElement('h1');
+    h1.innerText = 'MY BOOKS';
+    document.body.appendChild(h1);
+    const ul = document.createElement('ul');
+    for (const book in myBooksInfo) {
```

Comment:
I must admit that the instructions were not a 100% clear but the idea was to loop through the list of book id's (`myBooks`) and get the information about that book from the `myBooksInfo` object.
Like so:
```
for(const myBook in myBooks) {
    const myBookInfo = myBooksInfo[myBook];
    // .... rest of the code that creates the html using myBookInfo
}
```

Reply:
hi.
after couple of brain attack I understood that . it's so understandable, I don't know why it was so hard for me.
I changed my code according to your feedback. 
and I sow that here, for-in loop in arrays returns numbers. me shocked...